### PR TITLE
docs/user-guide/pod-security-policy: misc improvements

### PR DESCRIPTION
**Pull request by php-coder**
- update link
- improve wording
- update list of volumes

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2113)

### CODE REVIEW

**Comment by pweil-** [comment]
LGTM

File: docs/user-guide/pod-security-policy/index.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ into three categories:
 restrictive value. 
 - *Controlled by an allowable set*: Fields of this type are checked 
 against the set to ensure their value is allowed.
-- *Controlled by a strategy*: Items that have a strategy to generate a value provide
+- *Controlled by a strategy*: Items that have a strategy to provide
```
Comment (php-coder) [inline docs/user-guide/pod-security-policy/index.md]: @pweil- I've added another improvement ^^^. Let me know If I should squash commits.

Comment (pweil-) [inline docs/user-guide/pod-security-policy/index.md]: Yes, please squash.  Thanks

Comment (php-coder) [inline docs/user-guide/pod-security-policy/index.md]: Squashed.

**Pull request merged**